packages/runtime/src/types.ts

```diff
@@ -156,6 +162,10 @@ export interface Context {
   ) => DeployedAssetTxReceipt
   optIntoASA: (
     assetIndex: number, address: AccountAddress, flags: types.TxParams) => TxReceipt
+  deployApp: (
+    fromAccountAddr: string, flags: AppDeploymentFlags,
+    approvalProgram: string, clearProgram: string, idx: number
+  ) => DeployedAppTxReceipt
   addApp: (
```

Comment:
explained in above comment as well, we can simply replace addApp with `deployApp` in ctx. Sorry i should have seen in  the initial review. could you please update?